build.gradle

```diff
@@ -46,14 +46,10 @@ dlDepsDirS.mkdirs()
 //TODO: Add ProtocolSupport to dlLibsC (Obviously we're in beta and cannot download yet so compilation will have to be done for now)
 def dlDepsC = [
 	new Tuple(
-		'https://yivesmirror.com/files/spigot/spigot-1.12.2-R0.1-SNAPSHOT-b1609.jar',
-		'spigot-1.12.2-1.jar'
+		'https://mcmirror.io/files/Paper/Paper-1.13.2-b388.jar',
```

Comment:
This is likely to be incorrect by now. Should be archive.mcmirror.io/Paper/..., I think.